apps/api/v2/src/ee/bookings/controllers/bookings.controller.e2e-spec.ts

```diff
@@ -0,0 +1,205 @@
+import { bootstrap } from "@/app";
+import { AppModule } from "@/app.module";
+import { CreateReccuringBookingInput } from "@/ee/bookings/inputs/create-reccuring-booking.input";
+import { SchedulesRepository } from "@/ee/schedules/schedules.repository";
+import { SchedulesService } from "@/ee/schedules/services/schedules.service";
+import { AvailabilitiesModule } from "@/modules/availabilities/availabilities.module";
+import { PrismaModule } from "@/modules/prisma/prisma.module";
+import { UsersModule } from "@/modules/users/users.module";
+import { INestApplication } from "@nestjs/common";
+import { NestExpressApplication } from "@nestjs/platform-express";
+import { Test } from "@nestjs/testing";
+import { User } from "@prisma/client";
+import * as request from "supertest";
+import { UserRepositoryFixture } from "test/fixtures/repository/users.repository.fixture";
+import { withAccessTokenAuth } from "test/utils/withAccessTokenAuth";
+
+import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import {
+  getAllUserBookings,
+  handleNewBooking,
+  getBookingInfo,
+  handleNewRecurringBooking,
+  handleInstantMeeting,
+} from "@calcom/platform-libraries";
+import { ApiSuccessResponse, ApiResponse } from "@calcom/platform-types";
+
+describe("Bookings Endpoints", () => {
+  describe("User Authentication", () => {
```

Comment:
Having "User Authenticated" instead of "User Authentication" is clearer because we are not testing authentication but cases when a user is authenticated.